org.libvips.vipsdisp.json

```diff
@@ -0,0 +1,351 @@
+{
+    "app-id" : "org.libvips.vipsdisp",
+    "runtime" : "org.gnome.Sdk",
+    "runtime-version" : "41",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "vipsdisp",
+    "rename-desktop-file": "vipsdisp.desktop",
+    "finish-args" : [
+        "--device=dri",
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--talk-name=org.gtk.vfs.*",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=home"
```

Comment:
Generally speaking, if you switch your code to use GtkNativeFileChooser, it will automatically use file portal and you'll be able to drop these permissions.

Reply:
This is supposed to turn into the image view component for the nip2 rewrite for gtk4, so I'd like to be able to customize the file chooser.